evaluateRPN.js

```diff
@@ -32,13 +57,47 @@ function evaluateRPN(expr) {
       - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push
       - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop
   */
+  
+  // Create an array by splitting the input expresssion
+  let elements = expr.split(' ');
+
+  // Convert all numerals in the expression from strings into numbers
+  for (let i = 0; i < elements.length; i++){
+    if (Number(elements[i]) || Number(elements[i]) === 0){
+      elements[i] = Number(elements[i]);
+    }
+  }
+
+  // Handle the special case for a single number passed into the function
+  if (elements.length === 1){
```

Comment:
If you do the `Array.pop` / `Array.push` approach, you'll find this special case isn't necessary.